src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -799,19 +799,19 @@ class GrowableArray : public GrowableArrayWithAllocator<E, GrowableArray<E>> {
   }
 };
 
-// Leaner GrowableArray for CHeap backed data arrays, with compile-time decided MEMFLAGS.
-template <typename E, MEMFLAGS F>
+// Leaner GrowableArray for CHeap backed data arrays, with compile-time decided MemTag.
+template <typename E, MemTag F>
```

Comment:
Another parameter needing update, but shouldn't (can't?) be called `mem_tag` because of the
function parameter name for allocate().

Reply:
We can change this later in a followup.